installation_and_upgrade/ibex_install_utils/install_tasks.py

```diff
@@ -404,21 +409,34 @@ def backup_database(self):
             if task.do_step:
                 try:
                     mysql_bin_dir = self._get_mysql_dir()
-                    mysql_path = os.path.join(mysql_bin_dir, "mysql.exe")
-                    mysql_admin_path = os.path.join(mysql_bin_dir, "mysqladmin.exe")
-                    if not all([os.path.exists(p) for p in [mysql_path, mysql_admin_path]]):
-                        raise OSError
-                    if subprocess.call([mysql_path, "-u", "root", "-p", "--execute",
-                                        "SET GLOBAL innodb_fast_shutdown=0"]) != 0 or \
-                            subprocess.call([mysql_admin_path, "-u", "root", "-p", "shutdown"]) != 0:
-                        self._prompt.prompt_and_raise_if_not_yes(
-                            "Stopping the MySQL service failed. Please do it manually")
-                except OSError:
+                    RunProcess(SYSTEM_SETUP_PATH, "mysql.exe", executable_directory=mysql_bin_dir,
```

Comment:
Do these have to be run from the SYSTEM_SETUP_PATH? At this point we don't know if that path exists.